tests/src/ExtensionTest.php

```diff
@@ -6,7 +6,7 @@
 use PHPUnit_Framework_TestCase;
 use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
 use Symfony\Component\Config\Definition\Builder\NodeBuilder;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\Definition\ScalarNode;
```

Comment:
Good spot, it isn't failing on phpunit 5.2.9 but is on 5.2.12. I will log an issue against that. Ta.